pom.xml

```diff
@@ -88,23 +88,23 @@
 
         <!-- Dependencies -->
         <airline.version>0.7</airline.version>
-        <amqp-client.version>3.6.5</amqp-client.version>
+        <amqp-client.version>4.0.0</amqp-client.version>
```

Comment:
Has this been tested with a RabbitMQ server? I am hesitating a bit to upgrade to a new major version during beta.